resources/views/Reviews/show.blade.php

```diff
@@ -186,8 +188,8 @@
                         <span class="value">{{$review->review}}</span>
                     </div>
                 </div>
-                @if ($review->post_image !== null)
-                <img src="{{$review->post_image}}" alt="投稿写真">
+                @if ($review->store->post_image !== null)
+                <img src="{{$review->store->post_image}}" alt="投稿写真">
                 @else
                 <img class="post-img" src="/css/noimage.png" alt="NO_IMAGE">
```

Comment:
［SHOULD］
多分画像が表示されていないです。
```suggestion
<img class="post-img" src="../css/noimage.png" alt="NO_IMAGE">
```
で段階一個あげると表示されました。